### ISSUE TITLE
Witness in top 63 not being selected; Similar to -> #26 Witness block generation issue, seeing "broadcast_commit" and "broadcast_reveal" but not generating 

### ISSUE THREAD

**Issue by w-haleys-node-t** [Revolution-Populi/revpop-core#73]
Witness Node up and running 24/7 for 2+ days (w-haleys-node-t and I believe martinl420 has same issue). Both running on Ubuntu 18.04.6 (Bionic B), with full setup install, albeit on different host machines.

Each of these two nodes has enough votes to be in the top 63 potential (51 and 52 last I checked) witnesses so I would expect both of our nodes to be included in the top 21 multiple times over the last couple of days. Neither of us have been selected at all, akin to Issue #26, and I do not know whether it is because the 'main' 21 nodes generally chosen have a ridiculous amount of votes or something else. Obviously 'we' understand the votes being required to get into the top 63 but assume, after that, it should be random selection otherwise favouritism/money equals no chance for any other node starting up and 'just' getting in the top 63. I've also noticed, whether unrelated or not, that, of the 7200RVP/Hr distributed for block creation for the top 21 nodes, none actually seems to be paid, but 'disappears'?

Our nodes are pinging the blockchain, consistently, twice per hour. The first one, every hour, has the comment:

**'Commit Operation was send'**

The second, one states:

**'Reveal Operation was send'**

I hope that the information given helps this be fixed. Any information needed, please contact me.

Keep it up guys;)

PS I have also noticed, having sat and counted each node's blocks for an hour straight, that, on the start of the new hour (and supposed new witnesses) that, on the change, exactly the same witnesses were chosen and even in the same order!

**Comment by MartijnL4** [comment]
Hi guys, thanks for the explanation. Still cant manage to find the vesting balances item, with "main menu" do you mean the dashboard? I've tried to find it for a while now.

Keep up the good work. Had a blast setting this stuff up.

**Comment by RevPop** [comment]
It should be there under the menu drop down upper right - maybe try page refresh. Thanks!

**Comment by RevPop** [comment]
![image (1)](https://user-images.githubusercontent.com/69200709/141602704-cc8c223e-9ebf-4260-b31a-cc6419cebabb.png)

**Comment by MartijnL4** [comment]
I see now. Thanks!

**Comment by w-haleys-node-t** [comment]
Hi Guys;) Thanks for the update...we did think we were going a bit mad with us not turning up in the list. Obviously we're fumbling in the dark a bit trying to work out cause and effect with certain things so I/we apologise if anything seems obvious or stupid to you. Ultimately we're just trying to be 'ready' for mainnet and so are trying to understand how things work and are at a bit of a loss atm.

Our intention was to **a)** create nodes **b)** get votes to get into top 63 and **c)** get chosen as top 21 witnesses to **d)** see how it functions/where RVP goes etc. All so that we have an idea of how Mainnet will work.

We did the following a) got two nodes aforementioned functioning 24/7 (w-haleys-node-t and martinl420) b) got some votes from the community/each other which got us into space 51/52 c) As we saw there may be a functionality diff (as you confirmed) we tried to up our votes to get into top 21 (admittedly which we're sorry if this caused an issue, we didn't know it would at the time) but stopped when we realised we'd both drained the faucet and were never going to attain enough votes to get into the top 21 anyway..it would have taken 100 days of the same effort to acquire them. 

Whether by design (i.e. someone put us on the 'naughty step', lol) or something else (some functionality we do not understand - security protocol maybe) our nodes **both** then 'disappeared' from the available witness list. MartijnL4 created another user 'sorry-for-botting' and then linked that account to his node instead (that's now at pos 58). Even though my witness node is still working 24/7 it is not showing up on the Witness List nor as having any votes. Then, weirdly, last night at around 11pm (GMT) I got a msg from MartijnL4 with a png showing our witnesses had 'appeared' again out of nowhere...mine at 38 with 2,125,648 votes and his at 39 with 1,986,640. They then disappeared again by this morning. So, we honestly don't know what's going on/why and if there's anything we can do about it or not?

Finally, the 'Vesting' Balance you mentioned. I/we, having looked, are confused on the makeup of this/how it is related to our nodes activity? i.e. mine, W-Haleys-node-t, has 1,936 RVP. The Martinl420 node has, similarly, 1,937. These have been up more or less the same amount of time, as witness nodes, even though not picked as active witnesses, so 'could' be understood to be similar. However, the new one that he just set up days ago, has been up nowhere near the same amount of time yet has, 1,932 RVP. We don't understand how our nodes, obviously without understanding the process, acting (or trying to act as active witnesses) relates to our eerily similar Vesting Balance totals.

Cheers Guys:)
